crates/stages/stages/src/stages/headers.rs

```diff
@@ -134,11 +128,8 @@ where
             }
             last_header_number = header.number();
 
-            // Increase total difficulty
-            td += header.difficulty();
-
             // Append to Headers segment
-            writer.append_header(header, td, header_hash)?;
+            writer.append_header(header, U256::ZERO, header_hash)?;
```

Comment:
do we ever call this with something nonzero after this PR? if not then maybe we should just remove the arg?